galleries/examples/user_interfaces/embedding_in_qt_sgskip.py

```diff
@@ -44,18 +44,34 @@ def __init__(self):
         self._static_ax.plot(t, np.tan(t), ".")
 
         self._dynamic_ax = dynamic_canvas.figure.subplots()
-        t = np.linspace(0, 10, 101)
         # Set up a Line2D.
-        self._line, = self._dynamic_ax.plot(t, np.sin(t + time.time()))
-        self._timer = dynamic_canvas.new_timer(50)
-        self._timer.add_callback(self._update_canvas)
-        self._timer.start()
+        self.xdata = np.linspace(0, 10, 101)
+        self._update_data()
```

Comment:
```suggestion
        self._update_ydata()
```
Make the naming more consistent with `self.xdata`.